rts/lra/trip-controller/src/main/java/io/narayana/rts/lra/demo/tripcontroller/TripService.java

```diff
@@ -37,7 +37,8 @@ public void confirmBooking(Booking booking, WebTarget hotelTarget, WebTarget fli
 
         lraClient.closeLRA(new URI(booking.getId()));
 
-        // Note that earlier version of the quicksart returned the new booking status in the response to the closeLRA call,
+        // Note that earlier version of the quickstart returned the new booking status
```

Comment:
Personally I would remove this reference to the old way of the quickstart working and just describe how things are today but fine with me to keep it too.

Reply:
Agreed